manifests/b/BlueBubbles/BlueBubbles/1.12.0/BlueBubbles.BlueBubbles.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BlueBubbles.BlueBubbles
+PackageVersion: 1.12.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install
+Installers:
```

```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 680.)